hwdb.d/60-keyboard.hwdb

```diff
@@ -609,6 +609,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pn*[pP][aA][vV][iI][lL][iI][oO
  KEYBOARD_KEY_d9=unknown
  KEYBOARD_KEY_d2=sysrq                                  # Fn+Print = SYSRQ
 
+# Pavilion gaming 15-dk1* (lid keys)
+evdev:atkbd:dmi:bvn*:bvr*:bd*:br*:efr*:svnHP:pnHPPavilionGamingLaptop15-dk1*:*
+  KEYBOARD_KEY_d7=unknown
+  KEYBOARD_KEY_d8=unknown
```

Comment:
you have extra spaces, that is breaking the parser - each line here needs to be indented by one space only

Reply:
hello so I guess its fixed now can you review